### DISCUSSION TITLE
AI投入酒類研發🍺

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#101] [General]
**新聞大綱:**
日本知名的三寶樂啤酒公司 (Sapporo) 攜手日本 IBM，利用 AI 人工智慧來投入罐裝 Chu-Hi 的研發。只要輸入商品概念，AI 就會參考過去數據來**自動生成符合條件的食譜配方**，時間花費是過去的一半左右。三寶樂計畫在 2022 年內正式導入，並於 2023 年推出 AI 所構思的產品。
以往商品研發方式，有一大部分是必須仰賴開發人員，如果出現人事異動，知識技能出現斷層。而現今把數量約 1650 件的各項數據輸入當中，讓系統進行學習。包括食譜配方、人類味覺的評價資料，以及香料特徵。

**新聞心得:**
利用數據讓人工智慧學習並產生配方出來，不僅可以將多年累積的經驗、資料一併統整起來，更能用這種方式讓製造的時間大大減少且提高效率！而且啤酒公司使用這種技術感覺很少見？！會不會之後還能在數據上做調整後讓機器製成多樣的調酒器！喝起來🍺

**新聞來源:**
https://news.cnyes.com/news/id/4760767
![47909ccfae340aa8366404a2a8c5e76a](https://user-images.githubusercontent.com/90883526/140609009-63d6d34e-b1e8-4796-8c02-a442f79f828b.jpg)

**Comment by ccathyy** [comment]
如果之還能在數據上做調整，讓機器製成多樣調酒器的話，那我肯定直接買一台回家哈哈哈哈哈哈

**Comment by 110409538** [comment]
感覺很不錯，好好奇它的味道。

**Comment by minniewei** [comment]
挖啊!這樣在未來每個人都變成調酒師了!!!!

**Comment by yasklop** [comment]
好想喝喝看，感覺味道很不錯。

**Comment by ayakasanma** [comment]
我也很期待AI能夠調製出適合每個人的飲料！每次去酒吧都不知道點什麼QQ

**Comment by ghost** [comment]
雖然可以縮短釀造時間，可是這樣會不會反倒揠苗助長失去酒本身的風味呢?

**Comment by handsomeniko** [comment]
這樣就失去了釀酒師突發奇想的靈感了，人類的靈感我覺得是AI無法體現的。

**Comment by Shiba010** [comment]
不過有些創意感覺是ai很難發想的。

**Comment by jack90325** [comment]
這樣都不用專門去外面喝調酒，在家就可以自己做

**Comment by Lynn12378** [comment]
感覺很酷ㄟ希望未來有機會喝到AI做的酒

**Comment by qeeeee39** [comment]
Ai的分析方式給了人類很多靈感，不過事實上ai是從事物中歸類出的運算方法。這是不是說明了世界萬物都是一樣的運作原理。

**Comment by LanningYang** [comment]
AI數據上的分析，確實能給人類帶來方便，但少了手工溫暖
